devtools/scumm-md5.txt

```diff
@@ -489,7 +489,7 @@ freddi	Freddi Fish 1: The Case of the Missing Kelp Seeds
 	cf8ef3a1fb483c5c4b1c584d1167b2c4	-1	de	Windows	HE 73	-	-	Oncer
 	1f2e62b5a9c50589fc342285a6bb3a27	-1	he	Windows	HE 73	-	-	e_orz
 	507bb360688dc4180fdf0d7597352a69	26402	se	Windows	HE 73	-	-	Sven Arvidsson
-	df047cc4792150f601290357566d36a6	-1	us	All	HE 90	Updated	-	khalek
+	df047cc4792150f601290357566d36a6	-1	en	All	HE 90	Updated	-	khalek
```

Comment:
I don't understand this, are you sure that it is not US, but also GB? Do you have both? And same thing regarding further similar cases.